contracts/chain-adapters/Arbitrum_CustomGasToken_Adapter.sol

```diff
@@ -215,10 +215,11 @@ contract Arbitrum_CustomGasToken_Adapter is AdapterInterface, CircleCCTPAdapter
         address _l2RefundL2Address,
         IERC20 _l1Usdc,
         ICCTPTokenMessenger _cctpTokenMessenger,
+        uint32 _cctpDomainId,
         FunderInterface _customGasTokenFunder,
         uint256 _l2MaxSubmissionCost,
         uint256 _l2GasPrice
-    ) CircleCCTPAdapter(_l1Usdc, _cctpTokenMessenger, CircleDomainIds.Arbitrum) {
```

Comment:
I think we should pass in nativeTokenDecimals here and use it in _from18ToNativeTokenDecimals()

Reply:
Done https://github.com/across-protocol/contracts/pull/699/commits/6437f7d55cc9c274576350617b28cbb08bb65c10